src/main/java/org/sonarsource/sonarlint/ls/DiagnosticPublisher.java

```diff
@@ -44,7 +44,7 @@
 public class DiagnosticPublisher {
 
   static final String SONARLINT_SOURCE = "sonarlint";
-  static final String SONARQUBE_SOURCE = "sonarqube";
+  static final String REMOTE_SOURCE = "remote";
```

Comment:
Note: this will also need to be modified on the client in `hotspotsTreeDataProvider.ts`.